okio/src/jvmMain/kotlin/okio/Pipe.kt

```diff
@@ -56,21 +56,20 @@ class Pipe(internal val maxBufferSize: Long) {
         if (canceled) throw IOException("canceled")
 
         while (byteCount > 0) {
+          timeout.waitUntil(buffer) {
```

Comment:
Can you move the Pipe and Timeout changes to a different PR? You can use assumeFalse(windows) if there's tests that fail.

Reply:
PR for `Pipe` and `Timeout`: https://github.com/square/okio/pull/811/files

> You can use assumeFalse(windows) if there's tests that fail.

👍  Detecting that code is running on windows sounds like another rabbit hole but I'll try to do something. Given that these test are mostly JVM, `System.getProperty("os.name")` might work